eventdata/challenges/document_id_benchmark.json

```diff
@@ -1,350 +1,182 @@
 {% set p_bulk_indexing_clients = (bulk_indexing_clients | default(20)) %}
-{% set p_iterations = bulk_indexing_iterations | default(150000) %}
+{% set p_iterations = bulk_indexing_iterations | default(200000) %}
 {% set p_iterations_per_client = (p_iterations / p_bulk_indexing_clients) | int %}
 
 {
   "name": "document_id_evaluation",
-  "description": "Index about 30GB of data into five single shard indices using different document ID types. IDs are autogenerated by Elasticsearch, meaning there are no conflicts.",
+  "description": "Indexes about 40GB of data into seven single shard indices using different document ID types. IDs are autogenerated by Elasticsearch, meaning there are no conflicts.",
```

Comment:
In the README you mentioned disk space requirements. Do they need to be updated as well?

Reply:
Good catch. :-)